packages/xo-server-perf-alert/src/index.js

```diff
@@ -3,14 +3,18 @@ import { createSchedule } from '@xen-orchestra/cron'
 import { assign, forOwn, map, mean } from 'lodash'
 import { utcParse } from 'd3-time-format'
 
+const COMPARATOR_FN = {
+  '>': (a, b) => a > b,
+  '<=': (a, b) => a <= b,
```

Comment:
Is the equal really necessary?
Why can't we keep it simple and more like the other comparator?